src/algorithms/tonal/pitchyinfft.cpp

```diff
@@ -37,7 +37,7 @@ static Real _weightMask[] = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0,
 static const Real _weights[] = {-75.8, -70.1, -60.8, -52.1, -44.2, -37.5,
 	-31.3, -25.6, -20.9, -16.5, -12.6, -9.6, -7.0, -4.7, -3.0, -1.8, -0.8,
 	-0.2, -0.0, 0.5, 1.6, 3.2, 5.4, 7.8, 8.1, 5.3, -2.4, -11.1, -12.8,
-	-12.2, -7.4, -17.8, -17.8, -17.8}; // by default the original one is selected
+	-12.2, -7.4, -17.8, -17.8, -17.8}; // by default the original and custom one is selected
```

Comment:
This comment is a little fuzzy, maybe we can say something like "By default use custom weights designed specifically for the PitchYinFFT algorithm".